packages/conf-libgccjit/conf-libgccjit.1/opam

```diff
@@ -0,0 +1,64 @@
+opam-version: "2.0"
+maintainer: "Lukasz Stafiniak <lukstafi@gmail.com>"
+authors: [
+    "David Malcolm <dmalcolm@redhat.com>"
+]
+homepage: "https://gcc.gnu.org/wiki/JIT"
+license: "GPL-2.0-or-later"
+# On Debian systems, there is libgccjit0, but not libgccjit0-dev, so we need
+# to explicitly match the libgccjit-dev version to the gcc version.
+# On MacOS, both cc and gcc bind to the clang compiler, so the gccjit bindings
+# might not work out-of-the-box and we use a specific version of gcc below.
+available: [
+  ((os-distribution != "ol" & os-distribution != "oraclelinux") | os-version >= "9")
```

Comment:
We can move this later to an x-ci-accept-failure if anybody requires it on ol

Reply:
I would not exclude a whole distribution, if anyone still uses OL 8 they have the option to update. This case specifically is about old versions of OL which are misconfigured: the default gcc is older than the oldest available package for libgccjit.

Reply:
Yes, sorry I was too sloppy in my comment. I mean if somebody intends to use it in older ol, we can always keep it available and and let them compile their own gcc. In any case I don't think we have ol users at the moment

Reply:
Yes, sorry I was too sloppy in my comment. I mean if somebody intends to use it in older ol, we can always keep it available and and let them compile their own gcc. In any case I don't think we have ol users at the moment